docs/api/example-query.md

```diff
@@ -3,20 +3,22 @@ id: example-query
 title: Example Query - Making your First Call
 ---
 
-## Example query
+There are two ways of querying API. You can either use the cURL library, which is a low level developer's tool, or use Playground, an interactive editor from GraphQL API.
```

Comment:
I think this is slightly misleading, there are definitely lots of other ways to query the API. For production code, you would usually use a dedicated GraphQL client, not curl. Curl is just a way to play with the API from your command line.